### ISSUE TITLE
how to choose VALIDATION_INDICES ?

### ISSUE THREAD

**Issue by poincarelee** [visinf/irr#47]
Hi, 
   in sintel.py you fixed the VALIDATION_INDICES as follows:
     
![image](https://user-images.githubusercontent.com/38481402/160011005-60a7a46a-60ae-450a-aeaf-71b79156ceec.png)
  I was wondering if these values are chosen meaningfully.  Means five clips? What will happen if I want to choose indices randomly?

**Comment by hurjunhwa** [comment]
Hi,
I followed the train/val split from FlowNet2 (https://github.com/lmb-freiburg/flownet2/issues/141) for a fair comparison.
You can freely change if you want. If there are appearance overlaps between train and val split, it may not be easy to find a good early stopping point.

**Comment by poincarelee** [comment]
Thanks a lot. 
Seems that if much more data is used in the model, validation indices are hard to choose as fixed values due to the proportion.
 I was wondering if the choice of these indices were needed to keep as successive or more precisely part of a whole action.
Would it influence the performance of model when indices are chosen randomly?

**Comment by hurjunhwa** [comment]
In that case, How about using the k-fold cross-validation instead of using the fixed validation set?
In the end, what matters is to find a good stopping point that generalizes well to the test set, not to the validation set.

**Comment by poincarelee** [comment]
Ok, you are right.
